### DISCUSSION TITLE
New gustiness formulation

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/amwg_dev#509] [Discussion about coupled runs]
We have several runs to assess the new gustiness formulation: 
- 78b: old gustiness formulation 
- 81b: 3x (https://github.com/NCAR/amwg_dev/issues/507)
- 82b: 1x (https://github.com/NCAR/amwg_dev/issues/508)
- 83b: 0x (https://github.com/NCAR/amwg_dev/issues/510)
___

<img width="547" alt="Screen Shot 2024-03-15 at 3 10 31 PM" src="https://github.com/NCAR/amwg_dev/assets/9723220/2ee9cfda-66c2-4343-92b9-8d1d5c9e9199">

___

Contact: @cecilehannay, @megandevlan, @swrneale, @PeterHjortLauritzen, @adamrher, @JulioTBacmeister, @gustavo-marques

**Comment by cecilehannay** [comment]
@megandevlan, @swrneale: 
- how long do you want 81b and 82b
- I understood from rich at the meeting that 83b just need to be 30 years. Correct?

**Reply by megandevlan** [reply]
I'm thinking 81b and 82b should hit ~60 years; Rich, that's enough for more solid ENSO statistics right? 

Agree that 30 years should be enough for 83b.

**Reply by cecilehannay** [reply]
Hopefully, the Lab Sea not preparing to freeze in 81b. 
<img width="1111" alt="Screen Shot 2024-03-17 at 3 59 30 PM" src="https://github.com/NCAR/amwg_dev/assets/9723220/4bde4d30-f667-4b78-8c10-4ee9df1d0a43">

**Reply by cecilehannay** [reply]
It looks like that 81b froze over. 
<img width="968" alt="Screen Shot 2024-03-18 at 8 44 37 AM" src="https://github.com/NCAR/amwg_dev/assets/9723220/b0908b0a-9754-4130-8b30-064ce50444d1">

**Reply by megandevlan** [reply]
That’s definitely far from ideal. When I looked on Friday, the Lab Sea flux decreases were larger in 3X than the 1X case, so maybe that’s related somehow… I’ll take a look later today. It looks like even by year 25 or so, 81b is set on a colder track than the control or 1X (which is a little surprising, as 82b should be further from 78b).

**Comment by swrneale** [comment]
It is interesting that 81b gustiness magnitudes are approximately the same as the original implementation (64?) and that froze also, no?  I'm wonder @megandevlan whether you do a quick analysis of whether gustiness is actually happening locally around teh LS region? Maybe the different surface flux simulations could save us?, or 1.5x or 2x could be looked at?

**Reply by megandevlan** [reply]
I should also note, if we change the flux calculation, we might not need the gustiness at all. This is implemented for the Large & Pond-based flux calculation (current default). If we change to COARE (or UA), we wouldn't have the gustiness added. It might be worth a run with COARE active instead of LP+gusts at this point.

**Comment by megandevlan** [comment]
It looks like we don't actually output UGUST in these runs. But I can plot this in our AMIP runs to get an initial sense of things (below). One thing to note is that for the newer runs, I never changed the definition of UGUST as a variable. That is, for 1x and 3x then, UGUST is the magnitude of the gustiness, _not_ alpha*sqrt(Ugust**2). 

I added black contours for the levels 0, 0.025, 0.05, 0.1, 0.15 and 0.2; so the smallest contour is for 0.025 m/s (averaged over years 2-10). There's some activity for UGUST in all three cases but it intrudes much less into the Lab Sea in the 3x case than in the 1x or original gust formulations. 
![Screen Shot 2024-03-18 at 3 12 43 PM](https://github.com/NCAR/amwg_dev/assets/42625059/f74bd61b-47d6-453c-acf3-d9eb21697039)

**Reply by megandevlan** [reply]
I also have a quick look at U10 in each of the gust cases (all averaged for years 2-24 only), and as a difference from the control 078b. The difference plots (right-hand column below) are pretty odd, as I would've expected the smallest difference from the old gust case to be present in the 3x run. Instead, differences are are _largest_ there. Maybe there are some unintended feedbacks/teleconnections we haven't identified yet. 
![Screen Shot 2024-03-18 at 3 48 53 PM](https://github.com/NCAR/amwg_dev/assets/42625059/b82a2ae8-d47c-473b-acd1-6e08d6e52ec4)

**Reply by megandevlan** [reply]
Quick sanity check: the tropical response is more sensical for changes in U10. There are big tropical decrease for 0x, and the smallest decreases for 3x. So I think the multipliers are working just fine, but the non-local response is clearly complicated. 

![Screen Shot 2024-03-18 at 4 02 11 PM](https://github.com/NCAR/amwg_dev/assets/42625059/c32ac571-a21f-4acf-935c-b24f64d6c64f)

**Comment by megandevlan** [comment]
I ran the ADF for each of these new coupled tests vs. obs: https://webext.cgd.ucar.edu/BLT1850/b.e23_alpha16g.BLT1850.ne30_t232.082b/ (similar location for other runs). This is just years 2-30 to ensure that all the 8?b cases have a similar time period to average over, which also avoids the freeze over in 081b. 

In most fields I've looked at so far, the RMSE values are slightly lower with the new gust formulation than the old one, regardless of the multiplier. Especially for annual means, at least; it's a bit less clear looking seasonally. But for now, I'm interpreting this as an indication we can move forward with the new calculation (inside the square root) vs. the old (078b option). We should still re-check the ENSO in 1x now that we have 60 years though.
